src/Component/Atoms/Label.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Label from './Label';
+ 
```

Comment:
Include snapshot testing for every test. And also check the code coverage. Try to have the code coverage above 80.